### ISSUE TITLE
GPL-948 Automatic Creation of UAT Test Data for Beckman

### ISSUE THREAD

**Issue by andrewsparkes** [sanger/General-Backlog-Items#138]
**User story**
As R&D we would like to be able to generate test data for UAT testing runs of the Beckman methods. This includes the creation of test plates with specified numbers of pickable sample wells (order is important), the creation of those plates in the DART database where appropriate, the printing of plate barcode labels, and the creation of a cytomat plate hotel inventory file for those plates.

**Who are the primary contacts for this story**
Lesley S
Iraad B
Scoot T

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] input of a ordered list of plate definitions (with number of pickable wells required) (nice to have: use of existing partially picked sources via their barcode)
- [ ] creation of those plates in mongodb, mlwh and DART via run of crawler with a generated crawler file
- [ ] creation of printer labels file for use in Lighthouse UI (nice to have: optional choice for automatic printing of labels to choice of printer)
- [ ] creation of cytomat inventory file for use as input to Beckman control addition method

**Dependencies**
This story is blocked by the following dependencies:
#126 

**Additional context**
Goal is to create the test data required for a UAT simulated run of the Beckman methods.
For this we need a set of plates with specific numbers of pickable wells to be created in DART, we need barcode labels for those plates, and we need to recreate the inventory file that the cytomat input plate hotel would normally create to use in the Beckman control addition method that will in turn create the picking list for the cherrypicking method. It is important that the order of the plates in the inventory file is settable.
There is an existing python script we use to manually create crawler and printer files in confluence in the following link, but it does not create an ordered inventory file and is not suitable for R&D to run:
https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/Python+script+to+create+test+data
Preferable option is to embed a screen for the users within a UAT tab in Lighthouse-UI. We did discuss previously about embedding the test plate creation code in crawler itself, so maybe Lighthouse-UI could trigger a run of it there.

**Comment by andrewsparkes** [comment]
Superceded by stories https://github.com/sanger/lighthouse/issues/322 and https://github.com/sanger/lighthouse-ui/issues/73
